src/pykeen/models/baseline/utils.py

```diff
@@ -23,13 +23,33 @@ def get_csr_matrix(
     row_indices: numpy.ndarray,
     col_indices: numpy.ndarray,
     shape: Tuple[int, int],
+    dtype: numpy.dtype = numpy.float32,
+    normalize: bool = True,
```

Comment:
if you can imagine we ever want a norm besides l1, you could change this to be an  `Optional[str]` and have set to `"l1"` as default